File: Module 3 - Functions/parametres and Arguments/ReturnTypes/Multiplereturn.cpp
// Multiple return Statements
# include<iostream>
    using namespace std;

    string checkNumber(int n)
    {
    if(n > 0)
        return "Positive";
    else if(n < 0)
        return "Negative";
    else
        return "Zero";
    }

    int main(){
        
        cout <<checkNumber(10)<<endl;
        cout <<checkNumber(-1)<<endl;
        cout <<checkNumber(0)<<endl;

        return  0;
    }

File: Module 3 - Functions/parametres and Arguments/ReturnTypes/Multiplereturn.py
# Multiple return Statements
def checknumber(n):
    if n > 0:
        return "Positive"
    elif n < 0:
        return "Negative"
    else:
        return "Zero"

print(checknumber(10))
print(checknumber(-1))
print(checknumber(0))
